src/Liip/RMT/Command/InitCommand.php

```diff
@@ -70,6 +70,13 @@ protected function configure(): void
                 'choices_shortcuts' => array('g' => 'git', 'h' => 'hg', 'n' => 'none'),
                 'default' => 'none',
             )),
+            new InformationRequest('main-branch', array(
+                'description' => 'The default branch you want to use',
+                'type' => 'choice',
+                'choices' => array('main', 'master'),
+                'choices_shortcuts' => array('m' => 'main', 'mst' => 'master'),
+                'default' => 'main',
+            )),
```

Comment:
I don't like so much this question. If the user is saying `none` at the previous question, then it makes no sense to ask him a branch name.
But I have to admit that in most case users will use Git. Maybe allow also a choice `n=>none`. And adjust a bit the question:

    The default branch you want to use (select none if you are not using a VCS system)